### DISCUSSION TITLE
mapping e tabella 41

### DISCUSSION THREAD

**Discussion by annamaraspin** [italia/ansc#764] [Supporto servizi cooperativi] [answered]
Buongiorno,
nel mapping TRASCR_007 trovo questo campo obbligatorio

![image](https://github.com/italia/ansc/assets/127182190/849b2bb7-41a6-48eb-b0cf-e7f2a4812ebf)

fa riferimento alla tabella 41 dove l'utente dovrà precisare su quale mezzo di trasporto è avvenuta la nascita.

Esistono anche i casi di morte (MORTE_005 e 008) e di matrimonio (TRASCR_MATR_005) che si riferiscono ad eventi successi su treno, nave, etc però in questi mapping non viene richiesto di precisare il valore della tabella 41.

Come mai questa differenza?
Grazie

**Comment by mttfranci** [comment]
Salve, verificheremo la vostra segnalazione con gli esperti ANUSCA.

Saluti

Assistenza ANSC

**Comment by mttfranci** [comment, marked as answer]
Salve dall'analisi fatta non è necessario specificare il dato per gli altri casi d'uso indicati, mentre per la nascita viene registrato anche perchè il luogo di nascita fa parte dei dati identificativi dell'utente.

Saluti 

Assistenza ANSC
